types/entities.go

```diff
@@ -44,3 +64,11 @@ func (e *Entities) UnmarshalJSON(b []byte) error {
 func (e Entities) Clone() Entities {
 	return maps.Clone(e)
 }
+
+type implicitEntityUID struct {
+	EntityUID
+}
+
+func (i implicitEntityUID) MarshalJSON() ([]byte, error) {
+	return i.ImplicitMarshalJSON()
```

Comment:
I'd just move the body from ImplicitMarshalJSON into here directly. Can also get rid of the struct wrapper, and just do `type implicitEntityUID EntityUID`.

Reply:
Great suggestion!